src/api/web/endpoint/package.js

```diff
@@ -102,7 +101,8 @@ function addPackageWebApi(route: Router, storage: IStorageHandler, auth: IAuth,
       callback: function(err: Error, info: $SidebarPackage) {
         if (_.isNil(err)) {
           let sideBarInfo: any = _.clone(info);
-          sideBarInfo.latest = info.versions[info[DIST_TAGS].latest];
+          sideBarInfo.versions = convertDistRemoteToLocalTarballUrls(info, req, config.url_prefix).versions;
```

Comment:
Remember the `config.url_prefix` does not work with URL anymore, check https://github.com/verdaccio/verdaccio/pull/1299

Reply:
> Remember the `config.url_prefix` does not work with URL anymore with `4.0.0`, check #1299

I know, but I've made copy-paste from... https://github.com/verdaccio/verdaccio/blob/f0b8673ecfdc23b7f73a86a40b8a25f63367d32f/src/api/endpoint/api/package.js#L36